### DISCUSSION TITLE
Themen beschriften nicht am Ausdruck

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#39] [General]
Wir sollten meiner Meinung nach die Beschriftung in den Ausdruck aufnehmen.

Ähnlich Marker beschriften?

LG Christian

**Comment by gis-eni** [comment]
Ich nehme einmal an du meinst:
Das Beschriften, das der Anwender unter "Darstellung" bei den Werkzeugen anwerfen kann. Wird nicht in den Ausdruck übernommen.

Die Frage ist für mich: Wer verwendet das noch und gibt es nicht bessere Lösungen für dieses Werkzeug?
Eigentlich kommt dieses Werkzeug noch aus Zeiten vom ArcIMS und man konnte so, ohne die AXLs angreifen zu müssen Beschriftungen anbieten.

Diese Beschriftungen werden dann am Server gerendert, sprich, durch das Werkzeug werden die Renderer der MXD im AGS manipuliert/überschreiben. Das ist meiner Meinung nach nicht mehr zeitgemäß. Labeln ist ein rechen intensiver Prozess und durch falsche Anwendung kann der Server dann ziemlich in Schwitzen kommen und im schlimmsten Fall gar nicht mehr reagieren. 

Fazit: Wir sollten das Werkzeug eher los werden und nur im Ausnahmefall verwenden.

Besser: Beschriftungslayer in der MXD anfügen: 
Man kann ein Thema öfter in die MXD einbinden und dann immer anders beschriften. Dazu trenn man die eigentliche Darstellung von den Beschriftungen:
* Layer 1x einfügen zum Renderen
* Layer x mal einfügen und jeweils anders beschriften (ohne Darstellung)

Mit den Darstellungsvarianten kann ein Anwender dann zwischen diesen Beschriftungen (event mit Option-Boxes = immer nur eines gleichzeitig möglich) schalten.

Damit hat man die Beschriftung (Maßstäbe/Größe/...) in der Hand und alles landet auch im Ausdruck.

Fazit: Wird erst einmal nicht umgesetzt. Schauen wir lieber, dass wir diese "alten" Exoten los werden. Bring mehr Security und für den Anwender ist es ebenfalls durchgängiger (Sichtbarkeiten werden im TOC geschalten)
